src/Portal.js

```diff
@@ -62,7 +63,7 @@ let Portal = React.createClass({
     // Save reference for future access.
     if (overlay !== null) {
       this._mountOverlayTarget();
-      this._overlayInstance = React.render(overlay, this._overlayTarget);
+      this._overlayInstance = ReactDOM.render(overlay, this._overlayTarget);
```

Comment:
Not sure if this is in `v0.14.0-beta1` but it should be `ReactDOM. unstable_renderSubtreeIntoContainer` per https://github.com/react-bootstrap/react-bootstrap/blob/v0.24.5-react-pre.0/src/Portal.js#L55